modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/common/RasDetailsQueryParams.java

```diff
@@ -9,6 +9,8 @@ public class RasDetailsQueryParams {
     
     public static final String DETAILS_METHOD_QUERY_PARAM_VALUE = "methods";
 
+    public static final String[] SUPPORTED_DETAIL_QUERY_PARAMS = {DETAILS_METHOD_QUERY_PARAM_VALUE};
```

Comment:
We have the `SupportedQueryParameterNames` class - maybe we could re-use it somehow here since we're essentially doing a similar check?